### ISSUE TITLE
Request - TTT GOE Daggerless Option

### ISSUE THREAD

**Issue by SAKamjin** [emperorshammer/issues-incoming-queue#73]
Would it be possible to add an option to have the GOE display without the dagger. This used to be an option a long time ago to only show the neck medal and less the dagger off.

**Comment by SmTaUcM** [comment]
Yep, this one is an easy addition, I will add the option under the weapons tab for the next release.
Transfering this issue to the TTT3 repository
